### DISCUSSION TITLE
How to seed relational database with uuid

### DISCUSSION THREAD

**Discussion by mingzhe305** [planetscale/discussion#291] [FAQs] [answered]
Hello!

I am just trying to seed my database on Planetscale and followed the [tutorial](https://planetscale.com/blog/how-to-seed-a-database-with-prisma-and-next-js), but am unsure on the approach. Within my data.js file that I will be using to seed, how do I relate the uuid of topicID in topics model to topicID in questions model?

### **data.js (used for seeding):**
```
  const topics = [
    {
      topicName: 'P=V*I',
      topicLevel: 1,
    },
    {
      //default uuid of this topic to be passed into question
      topicName: "Ohm's Law",
      topicLevel: 2,
    },
  ];

  const questions = [
    {
      topicID; //uuid of Ohm's Law to be passed here
      questionContent: 'random content',
      questionDifficulty: 1,
    },
  ];
```

### **Topics model:**

```
model topics {
  topicID String @id @default(uuid())
  topicName String @db.VarChar(50)
  topicLevel Int
  topicID_rs questions[] @relation("topicID_rs")
}
```

### **Questions model:**
```
model questions {
  questionID String @id @default(uuid())
  topicID String
  questionContent String @db.VarChar(500)
  questionDifficulty Int //1-easy, 2-medium, 3-hard
  topicID_rs topics @relation("topicID_rs", fields: [topicID], references: [topicID])

  @@index([topicID])
}
```

**Comment by fooforge** [comment, marked as answer]
Apologies for not responding sooner! Looks like this has been answered in https://github.com/prisma/prisma/discussions/15510#discussioncomment-3737691 so going to mark this as resolved.
